src/Path.hs

```diff
@@ -103,22 +105,26 @@ parseJSONWith f x =
        Left e -> fail (show e)
 {-# INLINE parseJSONWith #-}
 
--- | Exception when parsing a location.
-data PathParseException
+-- | Exceptions during path operations
+data PathException
   = InvalidAbsDir FilePath
   | InvalidRelDir FilePath
   | InvalidAbsFile FilePath
   | InvalidRelFile FilePath
   | Couldn'tStripPrefixDir FilePath FilePath
```

Comment:
That was the first name I considered. But then I thought its not even a prefix, there is no question of valid or invalid. Nitpicking to the hilt :-) Any one of those should be fine.